setup.py

```diff
@@ -31,6 +31,7 @@
           "machomachomangler; sys_platform == 'win32'",
           "bindepend; sys_platform == 'win32'",
           "wheel",
+          "wheeltools >= 0.1.0b1",
```

Comment:
we can just write `wheeltools` in install_requires, once we publish a stable release